test/terra/noise/test_noise_transformation.py

```diff
@@ -212,15 +212,6 @@ def test_approx_noise_model(self):
 
         self.assertNoiseModelsAlmostEqual(expected_result, result)
 
-    def test_clifford(self):
```

Comment:
Instead of removing this test can we skip it with a skip message about the bug. Or is it not providing any value and we don't need the coverage?

Reply:
The good news is that cvxpy update 1.1.3 eliminated the problem, as far as I can see. The bad news is that the performance of the optimization engine is worse than before; I had to change the test's parameters in order to get something which works relatively well. But this is not a correctness issue.